_includes/v22.1/dedicated-pci-compliance.md

```diff
@@ -0,0 +1,9 @@
+{{site.data.alerts.callout_info}}
+{{ site.data.products.dedicated }} is certified by a Payment Card Industry (PCI) Qualified Security Assessor (QSA) for meeting the Payment Card Industry Data Security Standard (PCI DSS).
+
+{{ site.data.products.dedicated }} customers can achieve PCI DSS compliance by ensuring that any payments-related or other Personal Identifiable Information (PII) is encrypted, tokenized, or masked at source. This can be done either within the customer application or through a third-party intermediary solution, such as Satori.
+
+Please reach out to your Cockroach Labs account team to learn more about achieving PCI DSS compliance with {{ site.data.products.dedicated }}.
+{{site.data.alerts.end}}
+
+Learn more: [CockroachDB-Satori integration](../satori-integration.html)
```

Comment:
I think this is the wrong place. I think there should be a See Also section in the security overview instead. WDYT? Also, I don't think the link text is accessible. There is no such product as "CockroachDB-Satori" and yet it will read that way. I suggest "Integrate CockroachDB Dedicated with Satori" or something like that.

Reply:
agree.